.gitlab-ci/stages/prepare.yml

```diff
@@ -4,16 +4,52 @@
   extends: .base_register_stage
   stage: "Prepare"
   dependencies: []
+
+#
+
+Register new socialgouv/cdtn base image:
+  <<: *prepare_stage
+  variables:
+    CONTEXT: .
+    IMAGE_NAME: $CI_REGISTRY_IMAGE
+  only:
+    changes:
+      - yarn.lock
+      - package.lock
+      - packages/**/package.lock
```

Comment:
The base image contains build files too no ?

Reply:
only for data but you're right we need to add these file as a dependency